fiftyone/core/dataset.py

```diff
@@ -3301,13 +3296,11 @@ def _get_saved_view_doc(self, name, pop=False):
         return self._doc.saved_views[idx]
 
     def _validate_saved_view_name(self, name, skip=None, overwrite=False):
-        slug = fou.to_slug(name)
-
         for view_doc in self._doc.saved_views:
             if view_doc is skip:
                 continue
 
-            if name == view_doc.name or slug == view_doc.slug:
+            if name == view_doc.name:
```

Comment:
I don't' think this will work. Unique names will resolve to the same slug 
<img width="371" alt="image" src="https://user-images.githubusercontent.com/30936736/211690191-09989a1b-a947-495b-80b0-a339fb5ffeaf.png">

Reply:
Can you clarify? What is the issue?